.github/workflows/ci.yml

```diff
@@ -284,6 +284,22 @@ jobs:
           name: "${{ env.BUILD_DIR }}.tar"
           path: "${{ env.BUILD_DIR }}.tar"
 
+  build_runtime_windows:
+    needs: setup
+    if: needs.setup.outputs.should-run == 'true' && needs.setup.outputs.ci-stage == 'postsubmit'
+    runs-on: windows-2022
+    env:
+      BUILD_DIR: build-runtime
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # v2
+      - name: "Checking out runtime submodules"
+        run: bash ./build_tools/scripts/git/update_runtime_submodules.sh
+      - name: "Configuring MSVC"
+        uses: ilammy/msvc-dev-cmd@7315a94840631165970262a99c72cfb48a65d25d # v1.12.0
+      - name: "Building runtime"
+        run: bash ./build_tools/cmake/build_runtime.sh "${BUILD_DIR}"
```

Comment:
I think you could also add "shell: bash" to the job. https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-set-the-default-shell-and-working-directory